tests/compilers/traps.rs

```diff
@@ -37,6 +37,7 @@ fn test_trap_return() -> Result<()> {
     Ok(())
 }
 
+#[cfg(not(feature = "test-no-traps"))]
 #[test]
 #[cfg_attr(feature = "test-singlepass", ignore)]
```

Comment:
I think it would be better to ignore rather than not process the test (as well as target the native feature rather than `test-no-traps` which is more indirect):

```suggestion
#[cfg_attr(any(feature = "test-singlepass", feature = "test-native"), ignore)]
```